src/utils/processReaction.ts

```diff
@@ -53,6 +53,13 @@ export default async (discordData: ShowcaseDiscordData, internalData: ShowcaseDa
       log.error(`Could not delete submission post for ${project.name} (${project.id}): ${err}`)
       await safeSendMessage(channel, `⚠️ Could not delete project submission post. Please delete message ${project.id} manually. (Discord error)`)
     }
+
+    const projectSubmissionsChannelId = process.env.PROJECT_SUBMISSIONS_CHANNEL
+    if (projectSubmissionsChannelId) {
+      try { await (guild.channels.cache.get(projectSubmissionsChannelId) as TextChannel | undefined)?.bulkDelete(project.relatedMsgs) } catch (err) {
```

Comment:
Pretty long line, I see the push for brevity, but I think clarity should be prioritised here.
```suggestion
      try {
      	await (guild.channels.cache.get(projectSubmissionsChannelId) as TextChannel | undefined)?.bulkDelete(project.relatedMsgs)
      } catch (err) {
```

Reply:
Agreed. I feel like I ran a formatter or something that generated that but 🤷